modules/users/tests/server/user.server.routes.tests.js

```diff
@@ -742,7 +742,7 @@ describe('User CRUD tests', function () {
               }
 
               // Call the assertion callback
-              userInfoRes.body.message.should.equal('Username already exists');
+              userInfoRes.body.message.toLowerCase().should.containEql('username already exists');
```

Comment:
@mleanos : I have mentioned the versions but maybe you missed that comment:
D:\mongodb\bin>mongo.exe --version
MongoDB shell version: 3.3.1

D:\mongodb\bin>mongod.exe --version
db version v3.3.1
git version: e1a38c10c8e38f15af891ed330a31636f9e3857f
OpenSSL version: OpenSSL 1.0.1p-fips 9 Jul 2015
allocator: tcmalloc
modules: none
build environment:
distmod: 2008plus-ssl
distarch: x86_64
target_arch: x86_64